dependencies/influxdb/http.go

```diff
@@ -65,6 +70,18 @@ func (h HttpProvider) SeriesCardinalityReaderFor(ctx context.Context, conf Confi
 	}, nil
 }
 
+func (h HttpProvider) WriterFor(ctx context.Context, conf Config) (Writer, error) {
+	httpClient, err := h.clientFor(ctx, conf)
+	if err != nil {
+		return nil, err
+	}
+
+	service := apihttp.NewService(httpClient.Config.Host, "Token "+httpClient.Config.Token, apihttp.DefaultOptions().SetHTTPDoer(httpClient.Client))
```

Comment:
Is there a way we can contribute upstream so that this `Token mytoken` thing isn't so easy to mess up?

Reply:
Possibly - but I think it makes sense the way it is. This is a fairly low level of the client API and it is specifying the authentication header. The `NewClient` top level API handles the translation: https://github.com/influxdata/influxdb-client-go/blob/v2.3.0/client.go#L110